lib/migration-stream.js

```diff
@@ -75,7 +75,7 @@ function MigrationStream() {
       });
     } else {
       if (!v1TileDataLogged && process.env.LOG_V1_TILES) {
-        fs.writeFileSync(os.tmpDir() + '/v1-stats.json', JSON.stringify({ 'tileVersion': '1' }));
+        fs.writeFileSync(os.tmpdir() + '/v1-stats.json', JSON.stringify({ 'tileVersion': '1' }));
```

Comment:
just curious, what's this `D`--> `d` change about?

Reply:
Node.js v8 has a warning about `tmpDir` nearing deprecation, and that `tmpdir` should be used instead. Sorry I forgot to mention the change in the PR!

Reply:
And changing this reduces unnecessary logging for downstream applications

Reply:
🆒 🙇